### PR TITLE
LPS-196090 Maintain status when executing the update object entry action

### PR DESCRIPTION
**Pull request by devpedrotavares**
Related ticket: [LPS-196090](https://liferay.atlassian.net/browse/LPS-196090)

### CODE REVIEW

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/UpdateObjectEntryObjectActionExecutorImpl.java [hunk 1]
```diff
@@ -116,6 +121,11 @@ private void _execute(
 				new ObjectEntry() {
 					{
 						properties = values;
+						status = new Status() {
+							{
+								code = serviceBuilderObjectEntry.getStatus();
+							}
+						};
```
Comment (murilostodolni) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/UpdateObjectEntryObjectActionExecutorImpl.java:128]: What do you thing to inline the `serviceBuilderObjectEntry`? Something like this:
 ```java
new ObjectEntry() {
	{
		properties = values;

		setStatus(
			() -> {
				com.liferay.object.model.ObjectEntry
					serviceBuilderObjectEntry =
						_objectEntryService.getObjectEntry(
							primaryKey);

				return new Status() {
					{
						code =
							serviceBuilderObjectEntry.
								getStatus();
					}
				};
			});
	}
});
 ```

**Review by murilostodolni — CHANGES_REQUESTED** [review]
Hey @devpedrotavares 

Good job!! I finished my review. Let me know if you have questions.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/UpdateObjectEntryObjectActionExecutorImpl.java (hunk 1 above)
Comment (devpedrotavares) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/UpdateObjectEntryObjectActionExecutorImpl.java]: We can do that

**Comment by devpedrotavares** [comment]
ci:test:sf

**Review by murilostodolni — APPROVED** [review]
LGTM

**Comment by murilostodolni** [comment]
Hey @devpedrotavares 

Let wait for @marcelabc review do run the object and relevant suite, ok?

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
ci:forward:force

**Comment by marcelabc** [comment]
ci:forward:force

**Pull request closed without merging**